pages/review/[lesson].tsx

```diff
@@ -33,6 +33,10 @@ const Review: React.FC<QueryDataProps<AppData>> = ({ queryData }) => {
   const { lessons, session } = queryData
   const router = useRouter()
   const currentlessonId = router.query.lesson as string
+  if (!isLessonID(currentlessonId)) {
```

Comment:
There is a `validateLessonId` function that has already been made

https://github.com/garageScript/c0d3-app/blob/master/helpers/validateLessonId.ts

Reply:
hmm this is on the front-end though, so we shouldn't use the function i mentioned.

I believe all the `lessonIds` should be contained in the `session`, so you can use that in order to determine if the current route is a valid lessonId or not

Reply:
There was lesson info in session indeed. I removed unnecessary lessonID type.